data-processor/src/test/groovy/io/micronaut/data/model/query/builder/SqlQueryBuilderSpec.groovy

```diff
@@ -102,6 +102,52 @@ interface MyRepository {
         builder.buildInsert(annotationMetadata, entity).query == 'INSERT INTO sale (name,data,quantities,extra_data,data_list) VALUES ($1,to_json($2::json),to_json($3::json),to_json($4::json),to_json($5::json))'
     }
 
+    @Requires({ javaVersion >= 1.8 })
```

Comment:
I wasn't sure why the test above was `@Required({ javaVersion <= 1.8 })`, but my local computer is Java 17, so I made this >= 1.8. If you want me to switch it back to 1.8, just let me know. Or if you want me to remove the Requires, I can do that too.